client/state/data-layer/http/README.md

```diff
@@ -0,0 +1,52 @@
+# Raw HTTP API Layer
+
+[`wpcom-http`](../wpcom-http) issues requests to the WordPress.com API. However sometimes you will want to issue a request to another API, maybe even on a third-party host. This raw HTTP layer will allow you to do just that.
+
+It follows the same API than `wpcom-http`, the only difference from the user's perspective being the action dispatched.
+
+### Usage
+
+So, let's say you would like to do a `POST` request to https://api.example.com when the action `GET_EXAMPLE_DATA` is dispatched, later continuing with `GET_EXAMPLE_DATA_COMPLETED` for the data or `GET_EXAMPLE_DATA_ERROR` for the error. You'll be able to do that with the following code (that could be located under `third-party`):
```

Comment:
it would be good to encourage more semantic use of the response actions here than the typical pattern of `_COMPLETED` and `_ERROR` or `_SUCCESS` and `_FAILURE`.

Instead, maybe we could use an example where the successful response from the API injects data into the store but the failure simply notifies the user or resets some data in the store.

```md
…is dispatched, later continuing with `NORRIS_QUOTE_ADD` for the success or `NOTICE_CREATE` for the failure
```

or something like that.